components/molecule/buttonGroup/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@s-ui/react-molecule-button-group",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf ./lib && npx mkdirp ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "@schibstedspain/sui-atom-button": "1"
```

Comment:
this should be removed

Reply:
I need it to be able to do this → `import {atomButtonGroupPositions} from '@schibstedspain/sui-atom-button'` (added after @davidbarna review comment)

Reply:
It would be cleaner having the positions as props instead of this hard dependency